.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,72 @@
+# Preamble
+name: Deploy SaltProc docs
+
+on:
+  push:
+    branches:
+      - master 
+    paths:
+      - 'doc/**'
+      - 'saltproc/**'
+
+env:
+  CACHE_NUMBER: 0 #change to manually reset cache
+
+jobs:
+  deploy-docs:
+    runs-on: ubuntu-latest
+      #if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+      - uses: actions/checkout@v2
+
+# Setup conda 
+      - name: Set up conda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
```

Comment:
out of curiosity, is there a reason we're going with mambaforge? Is the dependency resolution not working out with others?

Reply:
Mambaforge uses mamba as the solver which is much faster than the default conda solver.

Reply:
I've definitely heard that. Can you add an inline comment documenting the choice so it's clear that's why we chose it?

Reply:
Absolutely!